src/driver/sap/SapConnectionOptions.ts

```diff
@@ -18,54 +18,21 @@ export interface SapConnectionOptions
     readonly schema?: string
 
     /**
-     * The driver objects
-     * This defaults to require("hdb-pool")
+     * Aborts communication attempts to the server
+     * after the specified timeout.
      */
-    readonly driver?: any
```

Comment:
Driver should not be removed. By default, we should initialize it with `require('@sap/hana-client')`. However, the user should be allowed to pass `require('hdb')` and the code should work with it.